README.md

```diff
@@ -21,6 +21,20 @@ Bluetooth Low Energy (BLE) library written with pure Node.js (no bindings) - bak
   - [GattService](https://github.com/chrvadala/node-ble/blob/main/docs/api.md#GattService)
   - [GattCharacteristic](https://github.com/chrvadala/node-ble/blob/main/docs/api.md#GattCharacteristic)
 
+# Pre-requisites
+This library works on many architectures supported by Linux. However Windows and Mac OS are [*not* supported](https://github.com/chrvadala/node-ble/issues/31).
+
+It leverages the `bluez` driver, a component supported by the following platforms and distributions <https://www.bluez.org/about>.
+
+*node-ble* has been tested on the following architectures:
+- Raspbian
+- Ubuntu
+- Debian
+
+```sh
+sudo apt install dbus bluetooth bluez libbluetooth-dev libudev-dev
```

Comment:
Are you sure that all these dependencies are needed? Usually I don't install them on ubuntu, but maybe that on debian something could be different.
Can you double check?

Reply:
Based on https://github.com/chrvadala/node-ble/issues/31#issuecomment-2054064653 - but its hard to tell with the setup I have it has a mix of docker and external debian packages that it uses

Reply:
Note that its not just Debian, its in a docker node:18-slim container so has any regular packages removed.